go.mod

```diff
@@ -13,41 +14,53 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/cobra v1.8.1
 	github.com/spf13/pflag v1.0.5
+	github.com/spf13/viper v1.18.2
 	github.com/stretchr/testify v1.9.0
 	github.com/thediveo/enumflag v0.10.0
-	github.com/wavesoftware/go-commandline v1.1.0
+	github.com/wavesoftware/go-commandline v1.3.0
 	github.com/wavesoftware/go-ensure v1.0.0
 	go.uber.org/zap v1.27.0
 	gopkg.in/yaml.v2 v2.4.0
 	gotest.tools/v3 v3.5.1
 	k8s.io/api v0.30.3
 	k8s.io/apimachinery v0.30.3
 	k8s.io/client-go v0.30.3
-	knative.dev/client/pkg v0.0.0-20241016014431-dbb49e603ce2
-	knative.dev/eventing v0.42.1-0.20241016013536-6aa49dc1db88
+	knative.dev/client/pkg v0.0.0-20241022160900-b3b6b8603082
+	knative.dev/eventing v0.43.0
 	knative.dev/hack v0.0.0-20241010131451-05b2fb30cb4d
 	knative.dev/pkg v0.0.0-20241021183759-9b9d535af5ad
 	knative.dev/reconciler-test v0.0.0-20241015093232-09111f0f1364
-	knative.dev/serving v0.42.1-0.20241021145327-c93d8c49a1b7
+	knative.dev/serving v0.43.0
 	sigs.k8s.io/yaml v1.4.0
 )
 
+// TODO: remove after knative/client#1973 is merged
+replace knative.dev/client/pkg => github.com/cardil/knative-client/pkg v0.0.0-20241025110936-4ece43045ffb
```

Comment:
/hold

https://github.com/knative/client/pull/1973 is required

Reply:
/unhold

https://github.com/knative/client/pull/1973 is now merged and integrated in this PR